Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,64 @@
+var botOptions = ["rock", "paper", "scissors"];
+
+var humanScore = 0;
+
+var computerScore = 0;
+
+function humanRock (e) {
+  var humanChoice = $("#rock").attr("id");
+  var botChoice = botOptions[Math.floor(Math.random() * botOptions.length)];
+  console.log(botChoice);
+  compare(humanChoice, botChoice);
+}
+
+function humanPaper (e) {
+  var humanChoice = $("#paper").attr("id");
+  var botChoice = botOptions[Math.floor(Math.random() * botOptions.length)];
```

Comment:
Perfect randomisation of `botChoice`! However, since you're repeating this line for all three functions, you might consider pulling this out into a separate function:

```
function getBotChoice() {
  var botOptions = ["rock", "paper", "scissors"];
  return botOptions[Math.floor(Math.random() * botOptions.length)];
}
```

By doing it this way you can also move the `botOptions` array inside the functions, as there's no real reason why that needs to be in a global variable. So then, you could use `getBotChoice()` from within the three wrapper functions by either assigning to a variable, or in line with my comment on line 8 above:

```
function humanPaper() {
  compare("paper", getBotChoice());
}
```